src/redisearch_rs/tools/shadow_tester/src/datasets.rs

```diff
@@ -0,0 +1,127 @@
+use std::path::PathBuf;
+
+/// Construct queries to create a dataset for the "1984" book from Project Gutenberg
+///
+/// This creates a part entry for every paragraph (separated by a blank line) in the book. These
+/// parts are then stored in the following structure:
+///
+/// ```
+/// 1984:part:<part_num> → {
+///   text: "paragraph content",
+///   part_num: M,      // Sequential numbering
+///   index_pos: 12345, // Character position in the full text
+///   part_len: 323     // Paragraph length
+/// }
+/// ```
+pub fn get_1984_index() -> Result<Vec<(String, Vec<String>)>, ureq::Error> {
+    let text = get_1984_text()?;
+
+    let mut part_num = 0;
+    let mut part_len = 0;
+    let mut index_pos = 0;
+    let mut part_text = vec![];
+
+    let mut setup_queries = vec![
+        ("flushall".to_string(), vec![]),
```

Comment:
What does this do/why do we need it?

Reply:
Since the `redis-server` does not shutdown correctly, this is to make sure the DB is clean on consecutive local runs. However, `flushall` actually does way to much and will interfere once we have other datasets too. I'll find queries to just clean up this dataset.